src/main/java/org/commcare/suite/model/DetailField.java

```diff
@@ -69,6 +70,16 @@
     public DetailField() {
     }
 
+    public String getPrintIdentifierRobust() {
+        // TODO: change this implementation once HQ work is done
```

Comment:
Ah, sorry I forgot about this. So part of the work Jenny is doing to enable this on HQ is to make it so that if printing is enabled for a detail, every `<field>` comes down with a `print-id` attribute that is equivalent to the key the app builder will use in the print template for that field. For now, just to prevent us from having to manually add a `print-id` to the suite file before that work is done, I was using this hack to grab the template argument if the print-id isn't there, since that usually (but not always) corresponds to what the `print-id` will be. Ideally we'd leave this here until Jenny's work is done, because otherwise it's going to be a pain in the ass for us to do testing in the interim...